drake/multibody/rigid_body_plant/visualization/contact_viz.py

```diff
@@ -0,0 +1,90 @@
+# note, this script runs in the main context of drake-visulizer
+# where many modules and variables already exist in the global scope
+
+from director import lcmUtils
+from director import applogic
+import numpy as np
+import drake as lcmdrakemsg
+
+
+class ContactVisualizer(object):
+    def __init__(self):
+        self.folder_name = 'Contact Results'
+        self.name = "Contact Visualizer"
+        self.enabled = True
+        # Subscribes to the LCM topic
+        lcmUtils.addSubscriber(
+            'CONTACT_RESULTS',
+            messageClass=lcmdrakemsg.lcmt_contact_results_for_viz,
+            callback=self.handle_message)
+
+    def is_enabled(self):
+        return self.enabled
+
+    def set_enabled(self, enable):
+        self.enabled = enable
```

Comment:
im going to recommend a small design improvement in order to make set_enabled() a little better:

move the call to `lcmUtils.addSubscriber` into a standalone method:

```
    def add_subscriber(self):
        self.sub = lcmUtils.addSubscriber(...)
        self.sub.setSpeedLimit(30) # if you want...

    def remove_subscriber(self):
        lcmUtils.removeSubscriber(self.sub)
        self.sub = None
```

Then call the add function from `__init__` and call the add or remove function from `set_enabled()`